superset-frontend/src/SqlLab/components/TableElement.jsx

```diff
@@ -126,11 +119,10 @@ class TableElement extends React.PureComponent {
       );
     }
     return header;
-  }
+  };
 
-  renderControls() {
+  const renderControls = () => {
     let keyLink;
-    const { table } = this.props;
     if (table.indexes && table.indexes.length > 0) {
```

Comment:
This can also be simplified to just `(table?.indexes)`

Reply:
Pushed up changes!